python/whylogs/experimental/core/udf_schema.py

```diff
@@ -182,6 +184,9 @@ def _run_udfs(
             self._run_udfs_on_dataframe(pandas, new_df, pandas.keys())
             new_df = pd.concat([pandas, new_df], axis=1)
 
+        if self.next_schema is not None:
+            new_df, new_columns = self.next_schema._run_udfs(new_df, new_columns)
+
```

Comment:
In its current form, this only chains the UDFs themselves--not the metrics, validators, etc. That's probably bad. All the schema bits should be merged, or we should split out a mechanism to just chain the UDFs alone.